content/en/digital_certificate.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+A digital certificate or  a public key certificate, which is issued by the Certificate Authority, is an electronic password that allows an entity to exchange data or resources over the internet in a secured manner. This is achieved by proving the authenticity of the user through the use of encryption methods and set of roles, policies, resources used to manage, use and revoke this digital certificate and encryptions. Much as a passport certifies one’s identity, the purpose of a digital certificate is to establish the identity of users within the ecosystem.
```

Comment:
```suggestion
Digital certificates or public key certificates are issued by a certificate authority (CA). These certificates are electronic files or passwords that allow entities to exchange data or resources over the internet securely. 

Certificates authenticate the user through encryption methods and a set of roles, policies, and resources used to manage, use and revoke certificates and encryptions. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservices](https://glossary.cncf.io/microservices/) app. 
```

Reply:
Edited the text a bit. Same content but worded slightly differently. This feels like a bit too much information: "through encryption methods and a set of roles, policies, and resources used to manage, use and revoke certificates and encryptions." It will probably raise more questions than answering them. Should we skip this? Maybe this is enough?

> Certificates authenticate users through encryption methods and a set of roles and policies. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservices](https://glossary.cncf.io/microservices/) app.